core/src/main/scala/org/bitcoins/core/psbt/PSBTRecord.scala

```diff
@@ -157,6 +151,11 @@ object InputPSBTRecord extends Factory[InputPSBTRecord] {
 
   case class WitnessUTXO(witnessUTXO: TransactionOutput)
```

Comment:
So is this defined in BIP174? and is it inherently insecure? If so should we even support it?

Reply:
Yeah it will still be included in BIP174 so I think it's best to continuing to support it.